CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.23
+
+###### _Jan 14, 2021_
+
+Big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Support public paths in module augmentation (#24267) @eps1lon
+- 👩‍🎤 Migrate the Paper to emotion (#24397) @povilass
+- 📚 Implemented various CI improvements (#24313) @eps1lon, (#24358) @oliviertassinari
+
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@material-ui/core@v5.0.0-alpha.23`/`@material-ui/unstyled@v5.0.0-alpha.23`
+
+- [Container] Fix disableGutters style not applied (#24284) @povilass
+- [Paper] Migrate to emotion (#24397) @povilass
+- [Slider] Allow mobile VO users to interact with Sliders (#23902) @CodySchaaf
+- [SwipeableDrawer] Add bleeding demo (#24268) @vicasas
+- [SwipeableDrawer] Fix overflow scroll computation logic (#24225) @yann120
+- [Table] Fix "more than" translation in es-ES (#24356) @vicasas
+- [Textfield] Add error color for form input with secondary color (#24290) @praveenkumar-kalidass
+
+### `@material-ui/lab@v5.0.0-alpha.23`
+
+- [DatePicker] Fix out of range month selection (#24301) @m4theushw
+- [DatePicker] Replace withDefaultProps with useThemeProps (#24309) @m4theushw
+- [DatePicker] Simplify ExtendWrapper type (#24275) @eps1lon
+- [DateRangePicker] Remove DateRangDelimiter (#24298) @huzaima
+- [TimePicker] Add pointer cursor for clock in desktop (#24276) @praveenkumar-kalidass
+- [lab] Drop usage of createStyles (#24158) @eps1lon
+- [lab] Fix import paths in generated declaration files (#24380) @eps1lon
+- [lab] Prevent possible null pointer in useValidation (#24318) @eps1lon
```

Comment:
This has a narrower impact, and is not about the lab package per say

```suggestion
- [DatePicker] Prevent possible null pointer in useValidation (#24318) @eps1lon
```

Reply:
These are still not only targetting the DatePicker.

Reply:
We could make them inside of [core] in this case, if you consider it's internal.

Reply:
It's not devops. It's about components in the lab.

Reply:
Ok so it's about the date pickers, at large?

Reply:
> Ok so it's about the date pickers, at large? 

Date and time pickers. Not just date pickers.

> I think that we should optimize for what the developers reading the changelog would want to know. 

We agree on that

> it seems more relevant to know that it's linked to a better date picker (and all its variants). 

A time picker is not a date picker.

Reply:
> A time picker is not a date picker.

I guess our disagreement comes from this, I was considering it close enough

Reply:
Should we then leave this as `lab` at this moment and merge? I'll start with the release right away

Reply:
Maybe we could create a new label that includes `DatePicker`, `TimePicker`, `DateRangePicker`, `DateTimePicker`, Like `Pickers`